src/main/java/be/minelabs/entity/BohrBlueprintEntity.java

```diff
@@ -132,9 +133,13 @@ private void logicalTick() {
         if (getAtomConfig().isElectronDecomposing()) {
             electronEjectProgress -= electronEjectProgressPerTick * getAtomConfig().getDecomposingElectronCount();
 
-            if (electronEjectProgress <= 0f){
-                if(removeItem(Items.ELECTRON))
-                    launchParticle(Items.ELECTRON);
+            if (electronEjectProgress <= 0f) {
+                if (removeItem(Items.ELECTRON))
+                    if (world.getGameRules().getBoolean(MinelabsGameRules.BOHR_PROJECTILES)) {
```

Comment:
Could move this check to the launchParticle function to make sure it is respected in the future as well (when this function is called from somewhere else maybe).